Output for first post mortem

giorgiosironi opened the pull request:
Doesn't need to have a perfect form, but if incorrect statements are made they should be corrected.

The corrective actions need to be agreed upon by the team.

Commit:
Add post mortem template

Commit:
Collect initial timeline to run first post mortem

Commit:
Add output of first post mortem

Commit:
Merge remote-tracking branch 'origin/develop' into add_post_mortem_template

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,79 @@
+
+All times UTC.
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads
```
@hdrury1 ... my recollection is that the whole thing was parked in favour of the playbook.

diversemix replied:
Also there was only one issue at this time.

hdrury1 replied:
My take: we agreed to park overwriting filenames in the knowledge that this would be more easily achieved after refactoring. There was a question around whether or not the files were supposed to upload sequentially and it was confirmed that this had always been the intention.

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,79 @@
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads
+
+10:42 ticket created by Hem to make files upload in parallel as a prerequisite for other tickets, in the Todo column
```
@damnhipster ... do you want to add "inadvertently" added to ToDo?

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,79 @@
+
+10:42 ticket created by Hem to make files upload in parallel as a prerequisite for other tickets, in the Todo column
+
+Peter understood that the file uploads issues were all to be parked, in favor of documentation
```
Probably should move this into the context of standup

hem-brahmbhatt committed:
Update 2019-02-06-multiple-file-uploads.md

hem-brahmbhatt committed:
Update 2019-02-06-multiple-file-uploads.md

diversemix commented:
PR's now need 2 reviewers, and `CONTIBUTING.md` has been updated see #1475

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,81 @@
+
+All times UTC.
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads being done beforehand
```
Still not happy with the wording of this - as parallel file uplaods did not exist as a concept/ticket until 10:42

hem-brahmbhatt replied:
I think Cory and I did mention it during the standup.

diversemix replied:
I've taken out the reason and just said `parked` ... followed by people's interpretation of what parked meant.

diversemix reviewed (dismissed):
~I think we need to be clear that parallel file uploads were not discussed at the standup.~

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,81 @@
+
+Inadvertently added in the Todo column
+
+Cory (and Hannah) thought that the work on file uploads will be large and postponed until further discussion.
```
This needs moving into the context of the standup and my name added here too.

diversemix commented on docs/incidents/2019-02-06-multiple-file-uploads.md:
```diff
@@ -0,0 +1,81 @@
+
+10:00 standup in which the overwriting file name issues is discussed and parked in favor of parallel file uploads being done beforehand
+
+Peter understood that the file uploads issues were all to be parked, in favor of documentation
```
I believe Hannah your understanding was that the documentation work was going to be done too.

diversemix committed:
Updated entry at 10:00

diversemix committed:
Add my name to the following intpretation as well

hdrury1 committed:
Updated write-up

To reflect own understanding and subsequent discussion amongst developers to decide upon corrective actions.
